imports/plugins/core/layout/client/components/footer.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import { registerComponent } from "/imports/plugins/core/components/lib";
+import { Reaction } from "/client/api";
+
+const Footer = () =>
```

Comment:
Always wrap arrow functions in `()` or `{}`.

Reply:
Done. I'm assuming you mean like this:
```
const Footer = () => (
  <div/>
);
```